src/test/java/net/minecraftforge/debug/PermissionTest.java

```diff
@@ -0,0 +1,42 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.commands.Commands;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.RegisterCommandsEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.server.permission.PermissionAPI;
+import net.minecraftforge.server.permission.events.PermissionGatherEvent;
+import net.minecraftforge.server.permission.nodes.PermissionNode;
+import net.minecraftforge.server.permission.nodes.PermissionTypes;
+
+@Mod("permissiontest")
+public class PermissionTest
+{
+
+   private static final PermissionNode<Boolean> blob = new PermissionNode<>("permissiontest", "test.blob", PermissionTypes.BOOL, (player, context) -> true);
+
+   public PermissionTest(){
+      MinecraftForge.EVENT_BUS.register(this);
+   }
+
+   @SubscribeEvent
+   public void registerNodes(PermissionGatherEvent.Nodes event)
+   {
+      event.addNodes(blob);
+   }
+
+   @SubscribeEvent
+   public void registerCommands(RegisterCommandsEvent event)
+   {
+      event.getDispatcher().register(Commands.literal("permissiontest")
+            .requires(src -> src.getEntity() != null && src.getEntity() instanceof ServerPlayer && PermissionAPI.getPermissions((ServerPlayer) src.getEntity(), blob))
+            .executes(context -> {
+               context.getSource().sendSuccess(new TextComponent("Blob"), false);
+
+               return 1;
+            }));
```

Comment:
It would be nice if you could extend the examples here.

Additionally besides a "positive" test, a "negative" test as well as examples with value types other then `Boolean` would be a good idea to add and test.

Reply:
The test mod has been expanded, is it ok now or would you like more?